resources/assets/js/label-trees/components/labelTreeLabel.vue

```diff
@@ -13,6 +13,10 @@
             </span>
             <span class="label-tree-label__buttons">
                 <button v-if="showFavourites" type="button" class="label-tree-label__favourite" :class="favouriteClass" @click.stop="toggleFavourite" :title="favouriteTitle">
+                    <span v-if="showFavouriteShortcuts" @click.stop="!toggleFavourite" >
```

Comment:
`toggleFavourite` is a method that should be called when the element is clicked. This does not make sense for the keyboard icon and number here, so please remove it.

You may have added this because a click on the keyboard should not remove the favorite label. To fix this, just move the `<span>` outside the `<button>` element.